assets/src/index.js

```diff
@@ -3,11 +3,15 @@ import ReactDOM from 'react-dom'
 import { BrowserRouter as Router } from 'react-router-dom'
 import './index.css'
 import App from './containers/App'
+import client from './service/client'
+import { ApolloProvider } from '@apollo/react-hooks'
 // import * as serviceWorker from './serviceWorker'
 
 ReactDOM.render(
   <Router basename='/'>
-    <App />
+    <ApolloProvider client={client}>
```

Comment:
is there a reason why `ApolloProvider` hook needs to apply at `index.js` instead of at the Assignment v2 level since for now it is only used at AssignV2. I don't see any harm here for doing it but trying to understand the rationale.

Reply:
I think it makes sense to put it at the top level of application, since I was planning on using GraphQL for other views like https://github.com/tl-its-umich-edu/my-learning-analytics/issues/839.

Reply:
I agree, I don't think there's much risk in using the `Provider` here, and it keeps us open to using GraphQL in other views later.